.github/workflows/build.yaml

```diff
@@ -9,3 +9,10 @@ jobs:
       - uses: actions/checkout@v2
       - name: Build
         run: mvn -B clean compile test
+      - name: Generate Code Coverage
+          run: mvn jacoco:report
```

Comment:
this indent is not like the others. Were these manually added, or by using ci_workflows ?